### PR TITLE
Link to Brett's talk from the communication page

### PR DESCRIPTION
**Pull request by Mariatta**
Will update with edited version of the video.
For now this should work I think :)
(I copied the text from the [talk abstract](https://www.pycascades.com/talks/setting-expectations-for-open-source-participation/))

### CODE REVIEW

**Comment by matrixise** [comment]
Hi Mariatta,

your PR seems fine for me, because we have the online video on youtube.

+1

Stephane

**Comment by matrixise** [comment]
Hi @willingc, it's a good idea to add the "Dial M for Mentor" +1

**Comment by Mariatta** [comment]
Thanks! I'll wait for @brettcannon to review and merge.

🙈  sure if any of you want to link Dial M for Mentor from the devguide somewhere ... (I don't know where)

File: communication.rst [hunk 1]
```diff
@@ -105,6 +105,13 @@ in return.
 
 .. _Diversity Statement: https://www.python.org/psf/diversity/
 
+Setting Expectations for Open Source Participation
+--------------------------------------------------
+
+Burn-out is common in open source due to a misunderstanding of what users, contributors,
+and maintainers should expect from each other. Brett Cannon gave a `talk <https://youtu.be/HiWfqMbJ3_8?t=7m47s>`_
+about this topic that will set out to help everyone set reasonable expectations of each other in
```
Comment (brettcannon) [inline communication.rst]: "that will set out" -> "that sets out"

**Comment by Mariatta** [comment]
Thanks! I made the suggested change :)

**Pull request merged**